test/extended/tbr_health/check.go

```diff
@@ -0,0 +1,16 @@
+package tbr_health
+
+import (
+	g "github.com/onsi/ginkgo"
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-devex][Suite:openshift/conformance/parallel] check registry.redhat.io is available and samples operator can import sample imagestreams", func() {
+	defer g.GinkgoRecover()
+	var (
+		oc = exutil.NewCLI("samples-health-check")
+	)
+	g.It("run sample related validations", func() {
+		exutil.WaitForOpenShiftNamespaceImageStreams(oc)
```

Comment:
for reference @bparees 

https://github.com/openshift/origin/blob/2863733fac5cdc382a0aac28c47b88359060578e/test/extended/util/framework.go#L233

Reply:
dumps the imagestreams and samples operator logs on failure, good.  that should be all we need to determine why the imagestreams are not present when this fails.

Reply:
doh! ... forgot to check for an error